src/components/Lights/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Dashboard from "../Dashboard";
+import Light from "../Light";
+import "../Lights/style.css";
+
+const Lights = ({ lights }) => {
+  return (
+    <>
```

Comment:
Na vice mist zbytecne pouzivas fragmenty (`<></>`). Neni to sice chyba ale je to neco navic. Fragmenty pouzivame pouze v pripadech kdy z komponenty vracime dva nebo vice prvku a nechceme je obalovat zadnym elementem jako je `<div>` ktery nema zadne styly a zadny vyznam.

Oficijani definice:

A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.

[Fragments dokumentace](https://reactjs.org/docs/fragments.html)